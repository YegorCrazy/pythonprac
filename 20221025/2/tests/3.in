import random
random.seed(42)

def inf_rand():
    while True:
        yield random.randint(0, 10)

it = slide(inf_rand(), 10)
print(*[next(it) for i in range(100)])
