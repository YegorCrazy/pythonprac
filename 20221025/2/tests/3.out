10 1 0 4 3 3 2 1 10 8 1 0 4 3 3 2 1 10 8 1 0 4 3 3 2 1 10 8 1 9 4 3 3 2 1 10 8 1 9 6 3 3 2 1 10 8 1 9 6 0 3 2 1 10 8 1 9 6 0 0 2 1 10 8 1 9 6 0 0 1 1 10 8 1 9 6 0 0 1 3 10 8 1 9 6 0 0 1 3 3 8 1 9 6 0 0 1 3 3 8
